Android/app/src/main/java/app/intra/net/go/GoIntraListener.java

```diff
@@ -88,6 +103,37 @@ public void onUDPSocketClosed(UDPSocketSummary summary) {
     event.putLong(Names.UPLOAD.name(), summary.getUploadBytes());
     event.putLong(Names.DOWNLOAD.name(), summary.getDownloadBytes());
     event.putLong(Names.DURATION.name(), summary.getDuration());
-    FirebaseAnalytics.getInstance(context).logEvent(Names.UDP.name(), event);
+    FirebaseAnalytics.getInstance(vpnService).logEvent(Names.UDP.name(), event);
+  }
+
+  private static final Map<Long, Status> goStatusMap = new HashMap<>();
+  static {
+    goStatusMap.put(Doh.Complete, Status.COMPLETE);
+    goStatusMap.put(Doh.SendFailed, Status.SEND_FAIL);
+    goStatusMap.put(Doh.HTTPError, Status.HTTP_ERROR);
+    goStatusMap.put(Doh.BadQuery, Status.INTERNAL_ERROR); // TODO: Add a Status
```

Comment:
nit: could you clarify what status you want to add?

Reply:
Done